src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -570,6 +571,14 @@ public function register(Container $app)
 
             $app['validator.validator_service_ids'] = array_merge($app['validator.validator_service_ids'], array('security.validator.user_password' => 'security.validator.user_password_validator'));
         }
+
+        if (class_exists('Symfony\Bundle\SecurityBundle\ArgumentResolver\UserArgumentResolver')) {
```

Comment:
SecurityBundle is not usable outside the fullstack framework, it depends on FrameworkBundle.
This class should probably be moved to a different location to make it reusable too